bindings/aws/s3/s3.go

```diff
@@ -178,9 +182,29 @@ func (s *AWSS3) create(ctx context.Context, req *bindings.InvokeRequest) (*bindi
 		return nil, fmt.Errorf("s3 binding error. Uploading: %w", err)
 	}
 
+	var presignURL string
+	if metadata.PresignTTL != "" {
+		d, parseErr := time.ParseDuration(metadata.PresignTTL)
+		if parseErr != nil {
+			return nil, fmt.Errorf("se binding error. Cannot parse duration %s: %s", metadata.PresignTTL, parseErr)
```

Comment:
typo: `s3 binding error:` (to be consistent with the other ones)

Reply:
Fixed, good catch